hw/hw_04_task_02.py

```diff
@@ -0,0 +1,12 @@
+import urllib.request
+from urllib.error import HTTPError, URLError
+
+
+# flake8: noqa: S310
+def count_dots_on_i(url: str) -> int:
+    try:
+        page = urllib.request.urlopen(url)
+        data = page.read().decode()
+        return sum(1 for symb in data if symb == "i")
+    except (HTTPError, URLError):
```

Comment:
Does URLError look like a network error?

Reply:
> URLError is raised because there is no network connection (no route to the specified server), or the specified server doesn’t exist.

 I think it looks like a network error, isn't it?